unstructured_platform_plugins/schema/json_schema.py

```diff
@@ -1,15 +1,16 @@
 import inspect
 from dataclasses import MISSING, fields, is_dataclass
-from enum import Enum, EnumMeta, EnumType
+from enum import Enum, EnumMeta
 from inspect import Parameter
 from pathlib import Path
 from types import GenericAlias, NoneType, UnionType
-from typing import Any, Optional, Type, Union, get_type_hints
+from typing import Any, Optional, Type, Union, _UnionGenericAlias
```

Comment:
The IDE shows it as not there because it's not meant to be a publicly used class, but the code should still run. It's because python doesn't have a notion of private items and really it's just a syntactic pattern.

Reply:
You're right, the thing is there. Strange that Jetbrains shows this as "cannot find ref" and refuses to even try looking for it.